stack.yaml

```diff
@@ -64,7 +64,7 @@ extra-deps:
   # a version > 1.0.0 of wai-middleware-prometheus is available
   # (required: https://github.com/fimad/prometheus-haskell/pull/45)
 - git: https://github.com/wireapp/saml2-web-sso
-  commit: 43c74bd44698e062abaa090c56de5272e66cee27  # master (Sep 24, 2020)
+  commit: dcbadb18f8ba6ca0e6245bf804405685d3e1cfce
```

Comment:
Can you add today's date here please? This helps us know how long it has been since and update quickly. While doing upgrades to LTS, it is very helpful.

Reply:
```suggestion
  commit: dcbadb18f8ba6ca0e6245bf804405685d3e1cfce  # master (Nov 12, 2020)
```